src/styles/container.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  margin: 0 auto;
+  padding: 40px 50px;
+  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($card-width, 1fr));
```

Comment:
Ensure that the variable `$card-width` is defined somewhere in your SCSS files. If it's not defined, it will cause an error during compilation.